### ISSUE TITLE
Automatically paste without popup window

### ISSUE THREAD

**Issue by DaveInCaz** [EslaMx7/PasteIntoFile#12]
An option to have it paste immediately as a file, using default settings, and without showing any window would be a nice improvement. Probably 99% of the time this would work fine once the defaults are set appropriately for the user.

Thanks for a cool tool!

**Comment by EslaMx7** [comment]
Thanks @DaveInCaz , I will make sure to have this in the next release soon.

**Comment by liwenjing93** [comment]
> Thanks @DaveInCaz , I will make sure to have this in the next release soon.

Thanks @EslaMx7 , I totally agree with @DaveInCaz , I hope you will release the next version soon including that feature.

**Comment by Mahrkeenerh** [comment]
@EslaMx7 Yeah, this feature would be a really nice thing to have, any progress by chance?

**Comment by MANONG571** [comment]
Excuse me, Does this feature come true?

**Comment by anwarulislam** [comment]
When we will be able to have this feature? Also, I found another tool called PasteHere. they have an inline renaming feature. which I found cool also. Can you do the same? which is relevant to this feature

![image](https://github.com/tomzorz/PasteHere/raw/master/demo.gif)
